.github/workflows/publish.yml

```diff
@@ -105,3 +105,14 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH }}
         run: npm publish --access public
+
+      - name: Create GitHub Releas
+        if: github.event_name == 'workflow_dispatch'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            await github.request(`POST /repos/${{ github.repository }}/releases`, {
```

Comment:
How do you feel using https://github.com/softprops/action-gh-release? I've found the `github-script` language to be very difficult to find documentation for and debug. Also, I think we need to include some files here for the release.

Reply:
Sounds good! I will give it a shot!

Reply:
@simlay which files do we want to upload along with the release? perhaps what's inside of the `dist` directory?

Reply:
Here b2e5b01 we have the change to use [softprops/action-gh-release](https://github.com/softprops/action-gh-release) instead